### PR TITLE
[doc] add mtl example and update get_gpu_family

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

documentation

## Description

add demo on MTL and update get_gpu_family

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: docs/weightonlyquant.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
 
 > Note: For LLM runtime model loading usage, please refer to [neural_speed readme](https://github.com/intel/neural-speed/blob/main/README.md#quick-start-transformer-like-usage)
 
-## Examples For Intel GPU
+## Examples For Intel GPU(PVC and ARC)
```
Comment (a32543254) [inline docs/weightonlyquant.md:149]: should we also add MTL after ARC ?

**Comment by a32543254** [comment]
please add mtl device in setup.py

**Review by a32543254 — APPROVED** [review]
LGTM

File: docs/weightonlyquant.md (hunk 1 above)
Comment (zhenwei-intel) [reply docs/weightonlyquant.md]: updated

**Comment by a32543254** [comment]
please also consider 
intel_extension_for_transformers/tools/utils.py, line 63, in get_gpu_family 
need to change the assert

**Comment by zhenwei-intel** [comment]
> please also consider intel_extension_for_transformers/tools/utils.py, line 63, in get_gpu_family need to change the assert

updated, thx

File: setup.py [hunk 2]
```diff
@@ -8,40 +8,12 @@
 from pathlib import Path
 from setuptools import Extension, find_packages, setup
 from setuptools.command.build_ext import build_ext
-
+from intel_extension_for_transformers.tools.utils import get_gpu_family
```
Comment (PenghuiCheng) [inline setup.py:12]: There is not itrex package in this time.

**Review by PenghuiCheng — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (zhenwei-intel) [reply setup.py:12]: absolute path import: intel_extension_for_transformers/tools/utils.py

**Pull request merged**